README.md

```diff
@@ -8,6 +8,20 @@ infrastructure.
 Setup
 -----
 
+### Setting up a GitHub PAT
```

Comment:
according to the actual contents of this step, I think it will be better/more accurate to rename this title to "Generating a Github token"